x-pack/plugins/uptime/public/components/monitor_management/monitor_list/enablement_empty_state.tsx

```diff
@@ -102,15 +105,15 @@ const MONITOR_MANAGEMENT_ENABLEMENT_MESSAGE = i18n.translate(
   'xpack.uptime.monitorManagement.emptyState.enablement',
   {
     defaultMessage:
-      'Enable Monitor Management to run lightweight checks and real-browser monitors from hosted testing locations around the world. Enabling Monitor Management will generate an API key to allow the Synthetics Service to write back to your Elasticsearch cluster.',
+      'Enable Monitor Management to run lightweight and real-browser monitors from hosted testing locations around the world. Enabling Monitor Management will generate an API key to allow the Synthetics Service to write back to your Elasticsearch cluster.',
```

Comment:
The term "Synthetics Service" is new to me so I'm not sure exactly what it refers to, but that might be ok if it's made clear in the linked document.

Reply:
@drewpost @paulb-elastic Have we thought about concrete terms for how we refer to the service that runs the tests on our hosted testing nodes?

Reply:
Good question...  I've been referring to it as the `Synthetics Cloud Service`.   @drewpost do you have something in mind?

Reply:
I'm guilty of sowing the seeds of confusion here. Internally we've called it synthetics service but as a part of the broader Synthetics product, this capability is "Global managed testing infrastructure"

Reply:
We have many places in docs where we use "Synthetics Service". I will be updated them all.